pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -430,3 +437,36 @@ def _detect_replacable_format_call(self, node: nodes.Const) -> None:
                 line=node.lineno,
                 col_offset=node.col_offset,
             )
+
+    @utils.only_required_for_messages("consider-refactoring-into-while-condition")
+    def visit_while(self, node: nodes.While) -> None:
+        self._check_breaking_after_while_true(node)
+
+    def _check_breaking_after_while_true(self, node: nodes.While) -> None:
+        """Check that any loop with an else clause has a break statement."""
```

Comment:
```suggestion
        """Check that any loop with an `if` clause has a break statement."""
```
I think it is meant to be `if`?

Reply:
```suggestion
        """Check that any loop with an ``if`` clause has a break statement."""
```